charts/mlsstats/templates/cronjob.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: {{ include "batch.apiVersion" . }}
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    app: mlsstats
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  concurrencyPolicy: Forbid
+  schedule: {{ .Values.schedule | quote }}
+  jobTemplate:
+    metadata:
+      labels:
+        app: mlsstats
+        release: {{ .Release.Name }}
+      # annotations:
+      #   # An annotation of the configmap checksum ensures changes to the configmap cause a redeployment upon `helm upgrade`
+      #   checksum/secret: {{ include (print .Template.BasePath "/secret.yaml") . | sha256sum }}
+    spec:
+      backoffLimit: 0
+      template:
+        spec:
+          restartPolicy: Never
+          # volumes:
+          #   - name: "ldap-scim-bridge-config"
+          #     secret:
+          #       secretName: {{ .Release.Name }}
+          containers:
+            - name: mlsstats
+              image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+              imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
+              args: [ "mlsstats"
+                      , "--cassandra-host", {{ .Values.cassandra.host | quote }}
+                      , "--cassandra-port", {{ .Values.cassandra.port | quote }}
+                      , "--cassandra-keyspace-galley", {{ .Values.cassandra.keyspaceGalley | quote }}
+                      , "--cassandra-keyspace-brig", {{ .Values.cassandra.keyspaceBrig | quote }}
+                      , "--cassandra-pagesize", {{ .Values.cassandra.pagesize | quote }}
+                      {{if .Values.s3.host -}}
+                      , "--s3-host", {{ .Values.s3.host | quote -}}
+                      {{- end -}}
+                      {{if .Values.s3.port -}}
+                      , "--s3-port", {{ .Values.s3.port | quote -}}
+                      {{- end -}}
+                      {{if .Values.s3.region -}}
+                      , "--s3-region", {{ .Values.s3.region | quote -}}
+                      {{- end -}}
```

Comment:
Can we just have defaults written down in values.yaml instead of these ifs. I think defaulting to real AWS endpoints would be good enough.